internal/driver/fetch.go

```diff
@@ -408,11 +408,13 @@ func locateBinaries(p *profile.Profile, s *source, obj plugin.ObjTool, ui plugin
 	}
 mapping:
 	for _, m := range p.Mapping {
+		cleanFile := strings.TrimPrefix(m.File, filepath.VolumeName(m.File))
```

Comment:
nit: I'd rename to `noVolumeFile` for clarity. "Clean" can mean many things.

Another nit: perhaps declare the var and handle its assignment in the same way as baseName and dirName which is under the condition? No need to handle this one differently.